doc/security.md

```diff
@@ -74,8 +75,9 @@ public class Application {
 As you see there are two arguments passed to the @EnableCustomSecurity:
 
 - authenticator: it introduces a class should be used for authentication.
-- ignoredPaths: using this parameter, you can specify the paths under which the authentication/authorization checks
-    are disabled.
+- applicationBasePathPattern: it is a required parameter that specifies the application's context path pattern.
```

Comment:
change description:
specifies the base path pattern under which security (authentication/authorization) is enabled. By default we will enable security on all paths: "/**"